guides/common/modules/con_performing-post-upgrade-tasks.adoc

```diff
@@ -1,6 +1,7 @@
 [id="performing-post-upgrade-tasks_{context}"]
 = Performing Post-Upgrade Tasks
 
-Optional: If the default provisioning templates have been changed during the upgrade, recreate any templates cloned from the default templates.
+- Optional: If the default provisioning templates have been changed during the upgrade, recreate any templates cloned from the default templates.
 If the custom code is executed before and/or after the provisioning process, use custom provisioning snippets to avoid recreating cloned templates.
 For more information about configuring custom provisioning snippets, see {ProvisioningDocURL}Creating_Custom_Provisioning_Snippets_provisioning[Creating Custom Provisioning Snippets] in _{ProvisioningDocTitle}_.
+- If you use custom templates, replace `@host.lifecycle_environment` with `@host.single_lifecycle_environment` and `@host.content_view` with `@host.single_content_view`.
```

Comment:
Technically, this doesn't apply to Foreman, only Katello/Orcharino/Satellite, but I am not sure it's worth the ifdef dance.